### ISSUE TITLE
Remove spaces from resource name output when exporting via direct invocation

### ISSUE THREAD

**Issue by smc87** [dynatrace-oss/terraform-provider-dynatrace#35]
https://www.terraform.io/docs/language/resources/syntax.html
Note: Resource names must start with a letter or underscore, and may contain only letters, digits, underscores, and dashes.

I currently get the below error when trying use the generated .tf files:
```
│ Error: Invalid resource name
│ 
│   on nonprod/dashboards/AMS General Monitor.dashboard.tf line 1, in resource "dynatrace_dashboard" "AMS General Monitor":
│    1: resource "dynatrace_dashboard" "AMS General Monitor" {
│ 
│ A name must start with a letter or underscore and may contain only letters,
│ digits, underscores, and dashes.
```

**Comment by smc87** [comment]
After removing the whitespace it seems terraform (v0.15) is complaining of further syntax issues:

```
│ Error: Unsupported block type
│
│   on nonprod\dashboards\AMSGeneralMonitor.dashboard.tf line 6, in resource "dynatrace_dashboard" "GROUP-IIB_and_MQStats":     
│    6:     filter {
│
│ Blocks of type "filter" are not expected here.
```

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hello @smc87,

Is it possible for you to share AMSGeneralMonitor.dashboard.tf with us?

thanks,
Reinhard

**Comment by smc87** [comment]
@Dynatrace-Reinhard-Pilz  of course. 

[AMS General Monitor.dashboard.tf.txt](https://github.com/dynatrace-oss/terraform-provider-dynatrace/files/6544996/AMS.General.Monitor.dashboard.tf.txt)

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Thanks alot. I'm hoping to get your issues sorted out by Friday when we release our next batch of resources. We'll keep this issue open until then.

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hi @smc87 
White Spaces will from now on get replaced with an underscore when exporting via direct invocation - not just for Dashboards but for all supported resources.

A couple of other minor issues came to light about Dashboards when working with your sample. They are all fixed now with v1.2.0.
